.github/workflows/checks.yml

```diff
@@ -23,6 +23,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16.15.1"
```

Comment:
Shouldn't this be the newest version that we switched to, `22.3.0`?

Reply:
Another good one I missed when trying different things. Corrected.